test/cmd.js

```diff
@@ -335,35 +381,6 @@ describe('express(1)', function () {
       .expect(404, /<h1>Not Found<\/h1>/, done);
     });
   });
-
-  describe('--help', function () {
```

Comment:
The diff is really confusing right now, so I'm not even 100% sure. Basically when I look at the diff on this pull request, I expect to see "add less tests" and not "add less tests and move a bunch of things around". It makes it much easier to accept the pull request when the only changes are exactly the description of the pull request and not sneaking in a bunch of other things.